JSONAPI/Json/JsonApiFormatter.cs

```diff
@@ -180,6 +180,13 @@ protected void Serialize(object value, Stream writeStream, JsonWriter writer, Js
         {
             writer.WriteStartObject();
 
+            //FIXME: The spec no longer requires that the ID key be "id":

```

Comment:
The more I think about it, to stick to the spec we probably shouldn't serialize it as "id" at all if that's not its name...but no option is clearly more appealing. I'm going to write both for now, but we'll probably revisit this, maybe the spec will change...again. (I liked it better when it simply required your ID to be "id".)